### PR TITLE
Fix dropdown scroll problem on touch devices

### LINKED ISSUE #2335: [BUG] Scrolling/swiping in dropdown on native devices with pull to refresh
- [ ] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
On native devices if a page has pull to refresh, sometimes pull to refresh is activated when swiping down in a dropdown popout list. Which can make it impossible to scroll/swipe down the list.

## Add any screenshots

https://user-images.githubusercontent.com/23026215/173568598-e29a73e6-303e-4e7b-a097-f20a53f1a432.mov

## Please complete the following information:
- OS: [iOS, Android]
- 
## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [x] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

### PR DESCRIPTION
**Pull request by mictro**
## Which issue does this PR close?

This PR closes #2335 

## What is the new behavior?

Prevent page refresh when dropdown is scrolled on touch devices

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [X] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [X] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: libs/designsystem/src/lib/components/dropdown/dropdown.component.ts [hunk 1]
```diff
@@ -236,7 +236,12 @@ export class DropdownComponent implements AfterViewInit, OnDestroy, ControlValue
     event.preventDefault();
   }
 
-  /* Utility that makes it easier to set styles on card element 
+  onTouchStart(event: TouchEvent): void {
+    // Prevent touchmove propgation to allow for scroll on page with pull-to-refresh;
+    event.stopPropagation();
+  }
```
Comment (RasmusKjeldgaard) [inline libs/designsystem/src/lib/components/dropdown/dropdown.component.ts:242]: Consider replacing this with the following snippet. 

I think it should result in identical behavior, and it uses a similar pattern to how we have done it for the keydown events in this component (bottom of this file). As an added bonus we should not need to add anything to the template.
 
```
@HostListener('touchstart', ['$event']) _onTouchStart(event: TouchEvent) {
  event.stopPropagation();
}
```

**Review by RasmusKjeldgaard — CHANGES_REQUESTED** [review]
Nice fix, I have proposed a minor refactor. Let me know what you think.

File: libs/designsystem/src/lib/components/dropdown/dropdown.component.ts (hunk 1 above)
Comment (mictro) [reply libs/designsystem/src/lib/components/dropdown/dropdown.component.ts]: Fixed

**Review by RasmusKjeldgaard — APPROVED** [review]
Very nice 👍

**Pull request merged**